samples/corda/corda-simple-application/scripts/initAsset.sh

```diff
@@ -8,3 +9,17 @@ CORDA_PORT=10009 ./clients/build/install/clients/bin/clients issue-asset-state 5
 echo -e "\n -- \n"
 echo "Adding 50 t1 tokens to PartyC"
 CORDA_PORT=10012 ./clients/build/install/clients/bin/clients issue-asset-state 50 t1
+
+if [ "$1" = "1" ]; then
```

Comment:
So `1` implies one Corda network is launched, otherwise both networks are launched?

Reply:
Yes.  The only usecase for this was workflow test. That's why I made it this way, so not to modify the existing docs.